src/databricks/labs/ucx/azure/locations.py

```diff
@@ -162,11 +162,22 @@ def _create_external_location(
         # if no credential found
         return None
 
+    def _rm_unsupported_location(self, location_urls: list[str]) -> list[str]:
```

Comment:
```suggestion
    def _filter_unsupported_location(self, location_urls: list[str]) -> list[str]:
```
To be consistent with filter() function

Reply:
fixed